loadtest/control/simulcontroller/websocket.go

```diff
@@ -29,6 +29,35 @@ func (c *SimulController) wsEventHandler(wg *sync.WaitGroup) {
 
 	for ev := range c.user.Events() {
 		switch ev.EventType() {
+		case model.WebsocketEventPosted:
```

Comment:
The code below is technically unrelated to the endpoint but I got here as a result of the investigation (see comment below). We can omit this change for now but it's something the web client is doing in https://github.com/mattermost/mattermost/blob/aa88f8bf59b8e43b6fd1aea35b8c4c81a6636738/webapp/channels/src/actions/post_actions.ts#L41